core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -43,6 +43,7 @@ import org.apache.kafka.common.record.Record
 import scala.collection.JavaConversions._
 import scala.collection.mutable.HashMap
 import scala.util.control.ControlThrowable
+import org.apache.kafka.clients.consumer.{ ConsumerConfig => NewConsumerConfig }
```

Comment:
I'll remove the extra spaces, just checked some other files and realized what format is being used. Thanks.